### ISSUE TITLE
Agaki fails to build - missing fusion.h

### ISSUE THREAD

**Issue by sempervictus** [hfiref0x/UACME#126]
Getting build errors on VS2019 using the current win10SDK:
![image](https://user-images.githubusercontent.com/1331084/164950377-f0467bb0-7fe5-4a75-be9a-d22b4267c946.png)

**Comment by hfiref0x** [comment]
https://github.com/hfiref0x/UACME#instructions
Install mentioned SDK. I've no idea what is the "current win10SDK" mean. They have multiple versions of win10 SDK and all of them can be installed at same time.

**Comment by FeCanuto** [comment]
i have this same issue, however i have Windows 10 (19041) SDK and v142 set correctly.

![image](https://user-images.githubusercontent.com/38158565/165416401-e428a34c-3739-467f-b0b9-86b52cf013fa.png)

i'm using a virtual studio 2022, any suggestion as to why this is happening?

**Comment by hfiref0x** [comment]
Okay, it seems MS moved this header from platform SDK to dotnet SDK.
Install NETFX SDK (4.8 for example) and try again.

**Comment by sempervictus** [comment]
Thank you very much, and sorry for the lag.
Will get the SDK in and retry.

**Comment by FeCanuto** [comment]
> Okay, it seems MS moved this header from platform SDK to dotnet SDK. Install NETFX SDK (4.8 for example) and try again.

This solution works for me, akagi compiled without losing fusion.h with NETFX SDK 4.8 installed.
Thank you hfiref0x.

**Comment by sakas23** [comment]
Could you please help me which one should i install from here? https://dotnet.microsoft.com/en-us/download/visual-studio-sdks 

I tried the 4.8 one but i could not compile the project. Thanks in advacned.
> > Okay, it seems MS moved this header from platform SDK to dotnet SDK. Install NETFX SDK (4.8 for example) and try again.
> 
> This solution works for me, akagi compiled without losing fusion.h with NETFX SDK 4.8 installed. Thank you hfiref0x.

**Comment by hfiref0x** [comment]
> Could you please help me which one should i install from here? https://dotnet.microsoft.com/en-us/download/visual-studio-sdks
> 
> I tried the 4.8 one but i could not compile the project. Thanks in advacned.

![fx](https://github.com/user-attachments/assets/051a631a-3995-412f-92f0-8f71d6df7e08)

**Comment by sakas23** [comment]
Thanks a lot. I finally managed to compile the project. I tried it on a Windows machine with OS Server 2016 and version 10.0.14393 Build 14393 ( A box from HacktheBox, I am practicing several concepts.) When i try several options like 33,61,67, the tool seems to work, but i still have the UAC prompt.  For other options i got nothing in return. Is there any chance something went wrong during the building stage? My akagi64.exe hash is  146418D041CBC1EA3F576FD003D8E018.  Thanks again for your time
